samples/scripts/externalclient-setup.js

```diff
@@ -141,23 +143,15 @@ async function main(config) {
   const auth = new GoogleAuth({
     scopes: 'https://www.googleapis.com/auth/cloud-platform',
   });
-
-  // TODO: switch to using IAM client SDK once v1 API has all the v1beta
-  // changes.
-  // https://cloud.google.com/iam/docs/reference/rest/v1beta/projects.locations.workloadIdentityPools
-  // https://github.com/googleapis/google-api-nodejs-client/tree/master/src/apis/iam
+  const authClient = await auth.getClient();
```

Comment:
You can skip this line entirely and say `google.options({auth});` and it will work just the same :)

Reply:
+1